app/frontend/src/views/About.vue

```diff
@@ -36,8 +44,10 @@
 
     <v-row justify="center" class="example-text">
       <v-col cols="12" lg="4">
-        <h2>{{ $t('trans.homePage.chefsHowToTitle') }}</h2>
-        <p>
+        <h2 :class="{ 'dir-rtl': isRTL }">
+          {{ $t('trans.homePage.chefsHowToTitle') }}
+        </h2>
+        <p :class="{ 'dir-rtl': isRTL }">
           {{ $t('trans.homePage.chefsHowToSub') }}
```

Comment:
It looks like the `chefsHowToSub` translations all contain "Get Started" at the end, but there is another `getStarted` text piece below that repeats it.

Reply:
(not related to this PR!)